robot/EDA/tests/browser/settings/edc_settings_page/edc_settings_page.robot

```diff
@@ -80,7 +80,7 @@ Verify EDA documents page is displayed when user clicks on the documentation but
     Current page should be                  Home        Education Cloud Settings
     Click product card button in edc home   Go to the EDA documentation.
     Switch Window    locator=NEW
-    Verify eda documentation  powerofus.force.com/s/article/EDA-Documentation   
+    Verify eda documentation  https://powerofus.force.com/s/article/EDA-Documentation   
```

Comment:
This is an old link. The new one is https://powerofus.force.com/EDA-Documentation

Reply:
@aarsvoboda This one is deliberately the same as it's in the robot test. What's going on here is it's waiting for the ultimate resolution of the url, and since it's a redirect, the updated one isn't the final destination, so we need to leave this as is.